bin/src/command/mod.rs

```diff
@@ -294,7 +294,12 @@ impl CommandServer {
     pub fn disable_cloexec_before_upgrade(&mut self) -> anyhow::Result<()> {
         for ref mut worker in self.workers.iter_mut() {
             if worker.run_state == RunState::Running {
-                util::disable_close_on_exec(worker.fd)?;
+                util::disable_close_on_exec(worker.fd).map_err(|e| {
```

Comment:
fix the warnings (by doing `let _ = util::disable_close_on_exec etc`) and it should be good to go

Reply:
done!